README.md

```diff
@@ -9,7 +9,10 @@ Install ros2 (needed for ament at least and ros2's C++ api for the version of rv
 https://github.com/ros2/ros2/wiki/Installation
 
 Whether you install from source or from a binary, there will be a `setup.bash` file you can source.
-Source that file before continuing:
+The current state of this branch requires a from-source build though.
+You might also need the following additional dependencies on Debian based systems: `liburdf-dev`, `libxaw7-dev`.
```

Comment:
`urdf` should be coming from here:

https://github.com/ros2/robot_model/tree/ros2/urdf

I think so anyways. I definitely didn't install anything like that on my macbook. I'm not sure about the `xaw` dep, it might be needed and getting pulled in automatically on my dev machine.

Reply:
Valid point. It was just not part of my local workspace but it is part of the ros2.repos file. So I removed it from this patch.